sdks/java/core/src/main/java/org/apache/beam/sdk/testing/PAssert.java

```diff
@@ -766,6 +830,16 @@ public PCollectionSingletonIterableAssert(
       return withPanes(GlobalWindow.INSTANCE, PaneExtractors.earlyPanes());
     }
 
+    @Override
+    public IterableAssert<T> filterAdditions() {
+      return null;
```

Comment:
The result is not nullable. Eventually (as soon as possible, if we can fund it) we will turn on a proper nullability analysis. This this code will cause work for whoever is doing that.